charts/harvester-csi-driver/templates/rbac.yaml

```diff
@@ -73,3 +73,6 @@ rules:
   - apiGroups: [ "storage.k8s.io" ]
     resources: [ "volumeattachments/status" ]
     verbs: [ "patch" ]
+  - apiGroups: [ 'policy' ]
+    resources: [ 'podsecuritypolicies' ]
+    verbs: [ 'use' ]
```

Comment:
same comment as above; this permission is too expansive for what harvester-csi-driver needs.